### PR TITLE
Workshops page

### LINKED ISSUE #5562: Create workshops page
Stakeholders: Vanesa, Fabiana
What? Create a landing page for workshop sign-ups
Why? Vanesa wants to create a page for Sourcegraph workshops so that bigger groups of people (ie: contributors of an OSS project) can quickly sign up for it. It's part of the onboarding process of OSS projects adopting Sourcegraph and is different from the sign-up for a demo page that we currently have as it would suggest a specific date to sign up for.

Assets:
[🔗Figma](https://www.figma.com/file/ISywg2RFakSm3kv8BYmXPV/Workshops%2FATC?node-id=0%3A1) → Same template as /[webinars](https://about.sourcegraph.com/webinars/code-insights-turning-your-metrics-into-action) 
📱Figma does not include a mobile version; it can be the same as webinars
🔗[Meta](https://docs.google.com/spreadsheets/d/1DakMmyE6bSG3BVChQoxn079VZJo6b-UCc59Q-J1zyJo/edit#gid=749641326)

🗒️Note that the form is a Demio form
Form embed code:
```
<script async id="demio-js" type="text/javascript" src="https://cdn.demio.com/production/dashboard/embed.bundle.js"></script> <span class="demio-embed-registration" data-hash="EPKkjeZ8F5wZT6sQ" data-api="api/v1" data-base-uri="https://my.demio.com/" data-form-width="100%" data-color="#5033E1
" data-text="REGISTER" ></span>
```
Context on Demio:
Demio is the webinar platform DG uses for AboveTheCode or any other webinar. Mate tested a Demio form once. They should embed well in our site (check [✓ [webinar] Verify we are able to embed a Demio form on our website](https://app.asana.com/0/1200095757619647/1201689377469179)

DG used to have a Demio page for sign-ups ([example](https://my.demio.com/ref/EPKkjeZ8F5wZT6sQ)), but we want to start hosting the page on our site

### PR DESCRIPTION
**Pull request by bretthayes**
Closes #5562 creating a workshops page for workshop sign ups.

### Test

1. Ensure prettier has standardized the proposed changes.
2. Ensure the form captures the data in HubSpot and you receive an email from the HubSpot workflow

### CODE REVIEW

File: src/pages/workshops/coreboot-july-2022.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { FunctionComponent } from 'react'
+
+import { Layout, GatedResourceLayout } from '@components'
+
+export const Workshops: FunctionComponent = () => (
+    <Layout
+        meta={{
+            title: 'Coreboot + Sourcegraph Workshop',
+            description: '',
```
Comment (st0nebraker) [inline src/pages/workshops/coreboot-july-2022.tsx:9]: If the description value is blank we can omit the whole line, it's not needed :)

**Review by st0nebraker — CHANGES_REQUESTED** [review]
It looks good to me! But I'm not seeing a HubSpot follow up email after form submission, are you @vanesa ? Just want to confirm the whole workflow is working. Do we need to do more steps on the HubSpot side of things?

**Review by vanesa — DISMISSED** [review]

**Review by st0nebraker — APPROVED** [review]

File: src/pages/workshops/coreboot-july-2022.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                    <p>Join us for a workshop on how Sourcegraph can make contributing to Coreboot easier.</p>
+                    <p> You'll learn:</p>
+                    <ul>
+                        <li> How to get access to Sourcegraph</li>
+                        <li> Different search types</li>
+                        <li> How to solve codebase challenges</li>
+                        <li> How to use code intelligence to navigate code efficiently</li>
+                        <li> How to do large-scale changes</li>
+                        <li> Documenting on Notebooks</li>
+                        <li> Code Insights</li>
```
Comment (bretthayes) [inline src/pages/workshops/coreboot-july-2022.tsx:39]: Trim spacing:

```suggestion
                    <p>You'll learn:</p>
                    <ul>
                        <li>How to get access to Sourcegraph</li>
                        <li>Different search types</li>
                        <li>How to solve codebase challenges</li>
                        <li>How to use code intelligence to navigate code efficiently</li>
                        <li>How to do large-scale changes</li>
                        <li>Documenting on Notebooks</li>
                        <li>Code Insights</li>
```

**Comment by bretthayes** [comment]
This is no longer needed. ([ref](https://sourcegraph.slack.com/archives/C023BV511TR/p1658511086831289?thread_ts=1658509475.151139&cid=C023BV511TR))

**Pull request closed without merging**